tox.ini

```diff
@@ -0,0 +1,20 @@
+[tox]
+skipdist = True
+envlist = py27,pep8
+
+[testenv]
+usedevelop = True
+deps =
+  -r{toxinidir}/requirements.txt
+  -r{toxinidir}/test-requirements.txt
+commands = py.test {posargs} --cov=simpleanidb --cov-config pytest.ini --cov-report term-missing \
+                             --cov-report html --junit-xml test-results.xml tests/unit
+
+[testenv:pep8]
+commands = flake8 {posargs}
+
+[flake8]
+ignore = H803,H301,H302,H802
+exclude = pyenv,.venv,.tox,dist,doc,*.egg,build,.ropeproject
+show-source = true
+max-line-length = 101
```

Comment:
only 80 is so 90s so 100 and you have to add one because the line break is counted as well